rules/repositories.bzl

```diff
@@ -112,6 +112,20 @@ swift_binary(
         """,
     )
 
+    if not native.existing_rule("xchammer"):
+        git_repository(
+            name = "xchammer",
+            commit = "792e31212bf244c1a4af6e1ecb5806c4f0eb64f2",
```

Comment:
We might need to create a tag for each of the versions we push into `rules_ios` - otherwise if we remove the fork or branches it could cause an error on the devs side

Reply:
Yeah, I'll have a branch in each fork and we can open PRs into those for now then bump the reference here, not super worried about upstreaming things yet as there's some work to be done first

Reply:
Btw re tags yeah makes sense, once I land this will create a tag in `master`, something like `xchammer-0.0.1`

Reply:
* https://github.com/bazel-ios/xchammer/releases/tag/rules_ios-0.0.1
* https://github.com/bazel-ios/tulsi/releases/tag/rules_ios-0.0.1